src/MessageContainer.js

```diff
@@ -134,33 +136,26 @@ export default class MessageContainer extends React.Component {
       <InvertibleScrollView
         {...props}
         {...invertibleScrollViewProps}
-        ref={component => this._invertibleScrollViewRef = component}
+        ref={component => (this._invertibleScrollViewRef = component)}
       />
     );
   }
 
   render() {
     const contentContainerStyle = this.props.inverted
-    ? {}
-    : styles.notInvertedContentContainerStyle
+      ? {}
+      : styles.notInvertedContentContainerStyle;
 
     return (
-      <View
-        ref='container'
-        style={styles.container}
-      >
+      <View style={styles.container}>
```

Comment:
Removing a `ref` will be a breaking change, do we really want to do this?

Reply:
I think in RN 0.50 has marked as deprecated. But yes, should be retro-compatible.

Reply:
At the very least it should be removed in a separate PR.

Reply:
EDIT: React 16

https://github.com/FaridSafi/react-native-gifted-chat/pull/643

Reply:
Ok :)